python/geoarrow-io/src/util.rs

```diff
@@ -1,6 +1,32 @@
+use pyo3::prelude::*;
 use pyo3_arrow::PyTable;
 
-pub(crate) fn table_to_pytable(table: geoarrow::table::Table) -> PyTable {
-    let (batches, schema) = table.into_inner();
-    PyTable::try_new(batches, schema).unwrap()
+/// A wrapper around a [PyTable] that implements [IntoPyObject] to convert to a runtime-available
```

Comment:
Note: we should update to the latest pyo3-arrow to get the primary impl of this: https://github.com/kylebarron/arro3/pull/269